utils.sh

```diff
@@ -161,6 +189,14 @@ function create_cluster() {
       jq -s '.[0] * .[1]' ${IGNITION_EXTRA} ${assets_dir}/worker.ign.orig | tee ${assets_dir}/worker.ign
     fi
 
+    if [ -n "${BOOTSTRAP_STATIC_IP}" ]; then
+      bootstrap_static_ip_ignition ${assets_dir}
+      mv ${assets_dir}/bootstrap.ign ${assets_dir}/bootstrap.ign.orig
+      jq '.storage.files += $input' ${assets_dir}/bootstrap.ign.orig --slurpfile input ${assets_dir}/bootstrap_network_config.ign | tee ${assets_dir}/bootstrap.ign
```

Comment:
FIXME - if we're going to run this in CI we probably want to remove the `tee` and redirect to avoid logging the ignition file